Engine/Main/GameObject.cpp

```diff
@@ -421,6 +421,9 @@ ENGINE_API Component* GameObject::CreateComponent(const Component::ComponentType
 	Component* created_component;
 	switch (type)
 	{
+	case Component::ComponentType::AABB:
+		created_component = new ComponentAABB(this);
+		break;
```

Comment:
This compoment is created automatically when creating `GameObject`, when are you using this case?